### PR TITLE
Add support for Toyota Corolla LE Hybrid 2020

### PR DESCRIPTION
**Pull request by hdoublearp**
Now working! Big shout out to @wocsor, couldn't have done it without you.

Will provide cabana logs for stock vs op in a few hours.

### CODE REVIEW

**Comment by thesoundofom** [comment]
Drive with OP: https://my.comma.ai/cabana/?route=09a3f95d3f19379f%7C2019-07-27--17-34-49&max=9&url=https%3A%2F%2Fchffrprivate-vzn.azureedge.net%2Fchffrprivate3%2Fv2%2F09a3f95d3f19379f%2F157ac83468eb4b9f9b9f2eb5a1d5c30a_2019-07-27--17-34-49

Drive with Stock LKAS: https://my.comma.ai/cabana/?route=09a3f95d3f19379f%7C2019-07-27--18-01-34&max=4&url=https%3A%2F%2Fchffrprivate-vzn.azureedge.net%2Fchffrprivate3%2Fv2%2F09a3f95d3f19379f%2Fe461d7090ecd797a36b33b07520ef084_2019-07-27--18-01-34

**Comment by rbiasini** [comment]
Looks great. Will review the drive soon.

File: README.md [hunk 1]
```diff
@@ -101,6 +101,7 @@ Supported Cars
 | Toyota               | C-HR 2017-19             | All                  | Yes     | Stock          | 0mph             | 0mph           | Toyota            |
 | Toyota               | Corolla 2017-19          | All                  | Yes     | Yes<sup>2</sup>| 20mph<sup>1</sup>| 0mph           | Toyota            |
 | Toyota               | Corolla 2020             | All                  | Yes     | Yes            | 0mph             | 0mph           | Toyota            |
+| Toyota               | Corolla Hybrid 2020      | All                  | Yes     | Yes            | 0mph             | 0mph           | Toyota            |
```
Comment (rbiasini) [inline README.md]: minor: alphabetically, this should go after the Hatchback

File: selfdrive/car/toyota/values.py [hunk 2]
```diff
@@ -182,6 +183,9 @@ def check_ecu_msgs(fingerprint, ecu):
...
   }],
+  CAR.COROLLAH_TSS2: [{
+    36: 8, 37: 8, 166: 8, 170: 8, 180: 8, 295: 8, 296: 8, 401: 8, 426: 6, 452: 8, 466: 8, 467: 8, 550: 8, 552: 4, 560: 7, 562: 6, 581: 5, 608: 8, 610: 8, 643: 7, 713: 8, 728: 8, 740: 5, 742: 8, 743: 8, 761: 8, 764: 8, 765: 8, 800: 8, 810: 2, 812: 8, 824: 8, 829: 2, 830: 7, 835: 8, 836: 8, 865: 8, 869: 7, 870: 7, 871: 2, 877: 8, 881: 8, 896: 8, 898: 8, 900: 6, 902: 6, 905: 8, 921: 8, 933: 8, 934: 8, 935: 8, 944: 8, 945: 8, 950: 8, 951: 8, 953: 8, 955: 8, 956: 8, 971: 7, 975: 5, 993: 8, 998: 5, 999: 7, 1000: 8, 1001: 8, 1002: 8, 1017: 8, 1020: 8, 1041: 8, 1042: 8, 1044: 8, 1056: 8, 1057: 8, 1059: 1, 1071: 8, 1076: 8, 1077: 8, 1084: 8, 1085: 8, 1086: 8, 1114: 8, 1132: 8, 1161: 8, 1162: 8, 1163: 8, 1164: 8, 1165: 8, 1166: 8, 1167: 8, 1172: 8, 1235: 8, 1279: 8, 1541: 8, 1552: 8, 1553: 8, 1556: 8, 1557: 8, 1568: 8, 1570: 8, 1571: 8, 1572: 8, 1592: 8, 1595: 8, 1649: 8, 1745: 8, 1775: 8, 1779: 8, 1786: 8, 1787: 8, 1788: 8, 1789: 8, 1904: 8, 1912: 8, 1990: 8, 1998: 8, 2015: 8, 2016: 8, 2024: 8
```
Comment (rbiasini) [inline selfdrive/car/toyota/values.py]: msgs 2014, 2015 and 2024 are related to VIN query. The rest of the fingerprint looks good though.

**Comment by rbiasini** [comment]
@thesoundofom : could you please let your EON to upload the video files as well for the route you posted?

**Comment by thesoundofom** [comment]
@rbiasini As far as I can tell everything looks to be uploaded now, I checked both and both the stock and OP routes are visible. Maybe the tail end didn't upload properly but 90% of it is up there... I'm in a very rural area with no internet. 

Also, I made the change to readme.md as requested.

@wocsor thoughts on removing 2015: 8, 2016: 8, 2024: 8 messages? This was the most complete print.

**Comment by rbiasini** [comment]
So, bad news: this fingerprint will conflict with the existing fingerprint for the Camry. We really need to move away from CAN based fingerprinting and switch to a VIN based fingerprint. Let's keep this PR open, but unfortunately it cannot be merged as is.

**Comment by ErichMoraga** [comment]
For now, anyone who needs their 2020 Corolla Hybrid LE working on OpenPilot 0.6.6 can switch to this branch on my fork (confirmed working by @Andreybeba today, as seen in Discord...
https://discordapp.com/channels/469524606043160576/524327905937850394/650879933945610260

Just SSH into your EON and run...
`cd /data && rm -rf openpilot && git clone https://github.com/ErichMoraga/openpilot && cd openpilot && git checkout 2020CorollaHybridLE && reboot`

**Comment by geohot** [comment]
If this pull request is still relevant, please resubmit it to the master branch. This should allow our development flow to work better, we commit to master, you commit to master, everyone commits to master! Learn more: https://medium.com/@comma_ai/a-2020-theme-externalization-13b33326d8b3

**Comment by thesoundofom** [comment]
No worries @geohot. I'm not sure if the Camry fingerprint issue is still current, but I can at least resubmit to master. Thanks

**Pull request closed without merging**